pkg/reconciler/route/route_test.go

```diff
@@ -272,6 +275,146 @@ func addResourcesToInformers(t *testing.T, ctx context.Context, route *v1alpha1.
 	fakeingressinformer.Get(ctx).Informer().GetIndexer().Add(ingress)
 }
 
+func TestHTTPChallengeRoute(t *testing.T) {
```

Comment:
Is it possible to have this test case as part of the table tests? I think this test case is a good fit.

Reply:
Sure, I'll give it a go.